src/Camille.RiotGames/gen/RiotGamesApiConfig.cs.dt

```diff
@@ -9,6 +9,17 @@ using System;
 
 namespace Camille.RiotGames
 {
+    public enum RegionConfig
+    {
+        /// <summary>The Region should be in the URL, as a subdomain</summary>
+        inUrlAsSubdomain,
+        /// <summary>The Region should be in the URL, as a query parameter</summary>
+        inUrlAsRegionQueryParameter,
+        /// <summary>The Region should be a header instead of in the URL</summary>
+        /// <seealso cref="RiotGamesApiConfig.RegionHeaderKey"/>
+        inHeader,
+    }
```

Comment:
enum value names should start with a capital letter
(and maybe it could be moved from `gen` to `src`)

Reply:
Thank you, and both items are now implemented.
I had missed that consideration in my thoughts of keeping it alongside the configuration variables themselves.